cypress/support/commands.js

```diff
@@ -23,3 +23,22 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+    cy.get(`[placeholder="${placeholder}"]`)
+  });
+
+Cypress.Commands.add('login', (username, password) => {
+    cy.request({
+      method: 'POST',
+      url: 'https://demoqa.com/Account/v1/Login',
+      body: {
+         userName: 'User1',
+         password: 'User12345!'
+        },
+    }).then((response) => {
+     cy.setCookie('token', response.body.token);
+     cy.setCookie('expires', response.body.expires);
+     cy.setCookie('userID', response.body.userId);
+     cy.setCookie('userName', response.body.username); 
+  });
+});
\ No newline at end of file
```

Comment:
Add a newline at the end of the file for better formatting and compatibility across different systems 📄